designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/FingerprintService.scala

```diff
@@ -16,26 +16,30 @@ class FingerprintService(dbioRunner: DBIOActionRunner, fingerprintRepository: Fi
     implicit ec: ExecutionContext
 ) extends LazyLogging {
 
-  def fingerprint(config: UsageStatisticsReportsConfig, fingerprintFileName: FileName): Future[Fingerprint] = {
+  def fingerprint(
+      config: UsageStatisticsReportsConfig,
+      fingerprintFileName: FileName
+  ): Future[Either[StatisticError, Fingerprint]] = {
```

Comment:
from the perspective of FingerprintService and its `fingerprint` method, we return a Fingerprint obj or information that the fingerprint cannot be generated (and IMO there is not need to say it's because of db error - it doesn't matter from the perspective of the client who uses it). So I'd not use `StatisticError` here

Reply:
Done